File: Android_Zone_Test/src/com/example/mylib_test/activity/custom_view/AndroidHeroActivity.java
package com.example.mylib_test.activity.custom_view;


import com.example.mylib_test.R;

import android.app.Activity;
import android.os.Bundle;

public class AndroidHeroActivity extends Activity{
	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		String temp = getIntent().getStringExtra("type");
		if("circle".equals(temp)){
			setContentView(R.layout.a_hero_circle);
		}else if("circle2".equals(temp)){
			setContentView(R.layout.a_hero_voice);
		}else{
			setContentView(R.layout.a_hero_scroll);
		}
		
	}

}

File: Android_Zone_Test/src/com/example/mylib_test/activity/custom_view/AndroidHeroActivity.kt
package com.example.mylib_test.activity.custom_view


import com.example.mylib_test.R

import com.zone.lib.base.controller.activity.BaseFeatureActivity

class AndroidHeroActivity : BaseFeatureActivity() {
    override fun setContentView() {
        val temp = intent.getStringExtra("type")
        if ("circle" == temp) {
            setContentView(R.layout.a_hero_circle)
        } else if ("circle2" == temp) {
            setContentView(R.layout.a_hero_voice)
        } else {
            setContentView(R.layout.a_hero_scroll)
        }
    }

    override fun initData() {
    }

    override fun setListener() {
    }
}
